timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -32,16 +33,16 @@ def __init__(self, index_name, sketch_id, timeline_id=None, **kwargs):
         self._rule = kwargs.get("rule")
         super().__init__(index_name, sketch_id, timeline_id=timeline_id)
 
-    def run_sigma_rule(
-        self, query, rule_name, tag_list=None, status_good=False
```

Comment:
Are there situations where you want to run a sigma rule against the timeline but not tag anything? `tag_list=None` seems to imply that it is an option but the description says the analyser will tag events. Also, there would be only the `ts_ttp` attributes added if the tag variable stays None. What should happen with stable sigma rules that have no `tag` field but hits in the events?

Reply:
so the tag list can be empty, in that case indeed only the file title would be applied to the event